css/src/main/scala/io/udash/css/CssText.scala

```diff
@@ -12,6 +12,6 @@ trait CssText {
 object CssText extends CssText {
   private final class TextStyleModifier(styles: CssStyle*) extends Modifier {
```

Comment:
`CssText#styles2TextMod` (varargs version) is not tested.

Aside question: is there even a possibility to use this implicit conversion?

Reply:
Only explicitly - removing both this one and the one in `CssView`.